viv-wallet-app/src/views/Wallet.vue

```diff
@@ -100,6 +89,17 @@ export default defineComponent({
             },
         },
     },
+    methods: {
+        formatConsultantStatus(status?: keyof typeof ConsultantStatus) {
+            if (status) {
+                return toString(ConsultantStatus[status]);
+            }
+            return "";
```

Comment:
maybe console warn ?

Reply:
done

Reply:
nice, but i think there's nothing interesting in `status`. but why not :)